packages/file/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+#### 6.1.2
+
+* `MemoryFileSystem` now provides `opHandle`s for exists operations.
+
 #### 6.1.1
 
-* `MemoryFile` now provides `opHandle`s for copy and open operations.
+* `MemoryFileSystem` now provides `opHandle`s for copy and open operations.
```

Comment:
I only added copy and open for `MemoryFile` since that's the only class that needed it. It wasn't a typo.

Reply:
I was reading this slightly differently as sort of "the library provides the ophandle class", but that isn't right either since this is talking about the exact type. I will change it back